pkg/actions/service.go

```diff
@@ -87,16 +88,17 @@ const defaultFetchSize = 1024
 
 var ErrNotFound = errors.New("not found")
 
-func NewService(ctx context.Context, db db.Database, source Source, writer OutputWriter, stats stats.Collector) *Service {
+func NewService(ctx context.Context, db db.Database, source Source, writer OutputWriter, stats stats.Collector, disabled bool) *Service {
```

Comment:
Consider a more descriptive name (and positive?) here, perhaps `executeActions`?

Reply:
Done.